src/Controller/ResultController.php

```diff
@@ -98,4 +102,23 @@ public function getResult(Request $request, array $requestAttributes): Response
             ]
         );
     }
+
+    /**
+     * Scores a specific quiz instance result.
+     *
+     * @param Request $request
+     * @param array $requestAttributes
+     * @return Response
+     */
+    public function scoreResult(Request $request, array $requestAttributes): Response
+    {
+        $quizInstanceId = $requestAttributes[self::QUIZ_INSTANCE_ID];
+        $score = $request->getParameter('score');
+        $this->quizInstanceService->saveScore($quizInstanceId, $score);
+        $body = Stream::createFromString('');
+        $response = new Response($body, '1.1', 301, '');
+        $response = $response->withHeader('Location', '/listResults');
```

Comment:
I will forever insist on creating a separate redirect method to use somewhere :D

Reply:
Did it. There is a separate PR for this :D